### DISCUSSION TITLE
Have you a ambassador program?

### DISCUSSION THREAD

**Discussion by Cornishon1** [pkgxdev/pkgx#770] [Help]
I want to help the project as an ambassador, so I wanted to ask about the ambassador program

**Comment by mxcl** [comment]
tea protocol will have such a program, please check their telegram: https://t.me/teaprotocol

**Comment by Cornishon1** [comment]
i am a ambassador NYM, you wanna AMMA with NYM?

вт, 3 жовт. 2023 р. о 23:44 Max Howell ***@***.***> пише:

**Comment by XynapseAI** [comment]
I also think this is necessary but the this time is not appropriate
